server/channels/api4/post.go

```diff
@@ -48,6 +48,26 @@ func (api *API) InitPost() {
 	api.BaseRoutes.Post.Handle("/move", api.APISessionRequired(moveThread)).Methods(http.MethodPost)
 }
 
+func postChecks(where string, c *Context, post *model.Post) {
```

Comment:
Continuing the discussion from the other thread, I think it'd be fine if we just rename this to `createPostChecks`. My problem with the current name is that I actually thought this would be called by other APIs as well when I first saw it

Reply:
Done.